src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const arrFromSource = sourceString
+    .trim()
+    .split(';')
+    .map((el) =>
+      el
+        .trim()
+        .split(':')
+        .map((item) => item.trim()),);
+
+  const obj = {};
 
+  arrFromSource.forEach((el) => {
+    if (el.length === 2) {
+      obj[el[0]] = el[1];
+    }
+  });
+
+  return obj;
```

Comment:
The function implementation is correct, but the variable naming could be improved for better readability and maintainability. Also, consider using `reduce` to create the object from the array of style declarations.